README.md

```diff
@@ -14,31 +14,50 @@ COBOL Language Support standardizes the communication between language tooling a
 ## Prerequisites
 
 - Java
-- (Optional) COBOL 3rd party plugin (required for syntax highlighting)
+- (Optional) COBOL 3rd party plugin (required for Syntax colouring)
 - Copybook path must be defined as **"COPYBOOKS"** folder in current workspace (required for copybook support)
 
-COBOL Language Support defines the protocol that is used between an editor or IDE and a language server that provides the following COBOL syntax awareness features:
+COBOL Language Support defines the protocol that is used between an editor or IDE, and a language server that provides the following COBOL syntax awareness features:
 
 - **Autocomplete**
 
-	- Provides suggestions for COBOL keywords while you type.
+	- Provides live suggestions while you type for:
+		- COBOL keywords
+		- COBOL variables
+		- COBOL paragraphs
+		- Code Snippet
 		<img src="Autocomplete.gif" alt="alt text" width="500px" height="300px">
 
-- **Syntax check for keywords**
+- **Syntax and Semantic Check for Code**
 
-	- Checks for mistakes and errors in COBOL code.
+	- Checks for mistakes and errors in COBOL code. Syntax check reviews and provides suggested fixes for the whole content of the code, through syntax and semantic analysis which returns diagnostics on the entire context of the code, not just keywords
+		
 		<img src="SyntaxCheck.gif" alt="alt text" width="500px" height="300px">
-- **Syntax highlighting** (with third-party plugins)
+		
+- **Syntax highlighting**
 	
-	- Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
+	- Enables syntax highlighting for COBOL code.
+	
+- **Syntax Coloring** (with third-party plugins)
```

Comment:
I think is better be uniform to uk or us language - if in this case we choose official English please correct this to colouring